app/models/lock_version.rb

```diff
@@ -2,6 +2,12 @@ class LockVersion < ApplicationRecord
   include Version
   belongs_to :target, polymorphic: true
 
+  after_save do
+    item = lock_version_target
+    return unless item
```

Comment:
I think usage of return here could have wider consequences than we expect. If I remember correctly (and I might not be) this would return from the method that is calling this block, whereas if we use next it will terminate this block.